lib/Netspoc/Approve/ASA.pm

```diff
@@ -174,6 +178,23 @@ sub get_parse_info {
                       { parse => \&check_int } ],
         },
 
+# ipv6 route if_name ipv6-prefix/prefix-length ipv6-address
+# [administrative-distance|tunneled]
+# Administrative-distance equals metric in IPv4 routing, handle it as in IPv4.
+# Tunneled is not provided for in IPv4, and likewise in IPv6.
+# IPv6 routing is stored under a different key to achieve seperated
+# IPv4 and IPv6 routing entries on device.
+        'ipv6 route' => {
+	    store => 'ROUTING6',
```

Comment:
Müssen wir wirklich zwischen ROUTING und ROUTING6 unterscheiden?
Können wir die beide Sorten zusammen unter ROUTING ablegen und danach einheitlich verarbeiten?

Reply:
Wenn wir am Ende zwei getrennte Blöcke für IPv4 und IPv6 routing haben wollen, ja: sub process_routing in Device.pm sortiert die Routingeinträge nach der Größe der Maske, und da kommen dann IPv4 und IPv6 Einträge durcheinander.